tpm.c

```diff
@@ -137,16 +102,14 @@ int tpm_extend_pcr(struct tpm *t, u32 pcr, u16 algo,
 	int ret = 0;
 
 	if (t->buff == NULL) {
```

Comment:
Ditch the brackets for single line in if statement.

Reply:
ack